app/controllers/internal/admin/print_controller.rb

```diff
@@ -34,6 +34,35 @@ def cover
         @barcode_for_html = Barby::HtmlOutputter.new(barcode)
         @barcode_as_string = barcode.to_s
       end
+
+      # Generates a table of contents from the given params[:old_folder_id]
+      def toc
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Cyclomatic complexity for toc is too high. [7/6]](https://www.codacy.com/app/lippertsjan/klausurtool-ror/file/2546221080/issues/source?bid=3141297&fileBranchId=3285107#l39)